pkgs/tools/nix/nixpkgs-manual-mmdoc/default.nix

```
@@ -0,0 +1,17 @@
+{ stdenvNoCC, mmdoc, nixpkgs-manual-lib-docs }:
+
+stdenvNoCC.mkDerivation rec {
+  name = "nixpkgs-minimal-manual";
+
+  src = builtins.filterSource (path: type: type == "directory" || builtins.match ".*\.md" path == [] || builtins.match ".*\.dot" path == []) ../../../../doc;
+
+  phases = [ "buildPhase" ];
+
+  buildPhase = ''
```

The other phases don't seem necessary here so use `buildCommand`, no need to `dontUnpack`
```suggestion
  buildCommand = ''
```